src/audio/multiband_drc/multiband_drc.c

```diff
@@ -235,9 +257,13 @@ static int multiband_drc_init(struct processing_module *mod)
 		return -EINVAL;
 	}
 
-	cd = rzalloc(SOF_MEM_ZONE_RUNTIME, 0, SOF_MEM_CAPS_RAM, sizeof(*cd));
-	if (!cd)
+	/* Memory allocation for multiband_drc_comp_data */
+	struct multiband_drc_comp_data *cd = rzalloc(SOF_MEM_ZONE_RUNTIME, 0,
+						     SOF_MEM_CAPS_RAM, sizeof(*cd));
+	if (!cd) {
+		comp_err(dev, "multiband_drc_init(), allocation for multiband_drc_comp_data failed");
```

Comment:
unneeded

Reply:
Could you please clarify if the removal of the `NULL `check after `rzalloc `is due to guaranteed error handling within the allocator, or are we following a specific coding standard that omits such checks?

Reply:
the change is unneeded (i.e. the trace), the check is still very much needed